### PR TITLE
Consistently use `Generic` as the last base class

### PR DESCRIPTION
**Pull request by sobolevn**
This is more like an experiment motivated by https://github.com/python/cpython/issues/106102

I want to see if it can change anything.

### CODE REVIEW

**Comment by AlexWaygood** [comment]
It's nice to be consistent about this as a matter of style, whether or not it has any semantic difference. We could maybe add a rule to flake8-pyi.

**Review by AlexWaygood — APPROVED** [review]

File: stubs/tqdm/tqdm/tk.pyi [hunk 1]
```diff
@@ -8,7 +8,7 @@ __all__ = ["tqdm_tk", "ttkrange", "tqdm", "trange"]
 
 _T = TypeVar("_T")
 
-class tqdm_tk(Generic[_T], std_tqdm[_T]):
+class tqdm_tk(std_tqdm[_T], Generic[_T]):
```
Comment (JelleZijlstra) [inline stubs/tqdm/tqdm/tk.pyi:11]: Technically the Generic base is redundant here, as there is another base that uses the TypeVar. That's another thing where maybe we could be more consistent.

Comment (JelleZijlstra) [reply stubs/tqdm/tqdm/tk.pyi:11]: (Though obviously no need to change that in this PR.)

**Comment by sobolevn** [comment]
Can we merge this? :)

**Comment by sobolevn** [comment]
Thanks everyone! 👏

**Comment by AlexWaygood** [comment]
> Can we merge this? :)

You're a maintainer! 😄

**Pull request merged**